### PR TITLE
[Modular] Adds moth milk

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
Adds a reagent called moth milk you can buy in the mothic vendor. It has no special effects. It simply tastes salty and can be added to other foods like a condiment.

## Why It's Good For The Game
image
Delicious moth milk.

## Changelog

:cl:
add: moth milk :)
/:cl:

### CODE REVIEW

**Review by projectkepler-ru — APPROVED** [review]

File: modular_zubbers/code/modules/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	color = "#F0E9DA" // rgb: 240, 233, 218
+	taste_description = "salty and oily substance"
+	ph = 6.5
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/moth_milk/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	if(!ismoth(M))
+		if(M.disgust < 80)
+			M.adjust_disgust(10 * REM * delta_time)
+		return TRUE
```
[the remark this answers is not in the document]
Comment (ReturnToZender) [reply modular_zubbers/code/modules/reagents/drink_reagents.dm]: Thank you so much for including what the code should look like in your suggestion, I've implemented this now

File: modular_zubbers/code/modules/reagents/drink_reagents.dm [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	description = "Whoever thought that milking moths is a good idea was totally wrong. Is it even milk?"
+	color = "#F0E9DA" // rgb: 240, 233, 218
+	taste_description = "salty and oily substance"
+	ph = 6.5
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/moth_milk/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	if(!ismoth(M))
+		if(M.disgust < 80)
+			M.adjust_disgust(10 * REM * delta_time,DISGUST_LEVEL_DISGUSTED)
```
Comment (BurgerLUA) [inline modular_zubbers/code/modules/reagents/drink_reagents.dm:12]: you forgot to remove the if(M.disgust < 80)

**Review by BurgerLUA — APPROVED** [review]
no other issues other than what I pointed out

**Comment by projectkepler-ru** [comment]
If there are no further objection this should get merged in 5 days at the latest

**Comment by StrangeWeirdKitten** [comment]
> If there are no further objection this should get merged in 5 days at the latest

I was meaning to do it, lol.

**Pull request merged**